docs/.c4/model.c4

```diff
@@ -0,0 +1,596 @@
+
+      // external relationship
+      external-tenant-loadbalancer -> istio-tenant-ingress-gateway
+    }
+
+    external-admin-loadbalancer = loadbalancer {
+      title 'External Admin LoadBalancer'
+      summary ' '
+      description 'Infrastructure defined load balancer for the admin gateway. Provides secure external access to UDS Core administrative interfaces like Grafana. Separated from tenant traffic for enhanced security and access control, typically restricted to administrative networks or VPNs.'
+
+      // external relationship
+      external-admin-loadbalancer -> istio-admin-ingress-gateway
+    }
+
+    external-services = infraServicesGroup {
+      title 'External Infrastructure Services'
+      summary ' '
+      description 'External infrastructure services supporting UDS Core components. These services run outside the Kubernetes cluster and provide persistent storage, databases, and other infrastructure dependencies required by UDS Core. Typically managed by cloud providers or infrastructure teams.'
+
+      ext-keycloak-db = infraService {
+        title 'External Keycloak Database'
+        summary ' '
+        description 'External PostgreSQL database storing Keycloak configuration and user data. Hosts Keycloak realms, clients, users, roles, and session information. Must be highly available and backed up regularly as it contains critical identity and access management data. Accessed via the Keycloak waypoint proxy over SSL/TLS on port 5432.'
+      }
+
+      ext-loki-storage = infraService {
+        title 'External Loki Storage'
+        summary ' '
+        description 'External object storage (S3-compatible) for Loki log data. Stores compressed log chunks and indexes for long-term retention. Supports AWS S3, MinIO, Azure Blob Storage, or other S3-compatible backends. Provides cost-effective, scalable storage for high-volume log data with configurable retention policies.'
+      }
+
+      ext-velero-storage = infraService {
+        title 'External Velero Storage'
+        summary ' '
+        description 'External object storage (S3-compatible) for Velero backups. Stores cluster backups including Kubernetes resource manifests and persistent volume snapshots. Critical for disaster recovery and cluster migration scenarios. Should be in a different failure domain than the cluster for true disaster recovery protection.'
+      }
+    }
+  }
+  // personas
+  admin = actor {
+    title 'Cluster Administrator'
+    summary ' '
+    description 'Administrator managing and monitoring the UDS Core platform. Platform operators and SREs who deploy, configure, and maintain UDS Core. Access administrative interfaces like Grafana for monitoring, Prometheus for metrics analysis, and Keycloak for identity management. Authenticate via Keycloak SSO with elevated privileges.'
```

Comment:
```suggestion
    description 'Administrator managing and monitoring the UDS Core platform. Platform operators and SREs who deploy, configure, and maintain UDS Core. Access administrative interfaces like Grafana for monitoring and Keycloak for identity management. Authenticate via Keycloak SSO with elevated privileges.'
```

Since Prometheus UI isn't exposed 🫣